project.clj

```diff
@@ -69,7 +69,7 @@
             [lein-auto "0.1.2"]
             [lein-less "1.7.5"]
             [lein-shell "0.5.0"]
-            [cider/cider-nrepl "0.15.0-SNAPSHOT"]
+            [cider/cider-nrepl "0.16.0"]
```

Comment:
Shouldn't Cider be removed from here instead, as explained [here](https://github.com/status-im/open-bounty/pull/199#discussion_r158851682)?

Reply:
Didn't know about that discussion! I agree then - I also put CIDER dependency ~/.lein/profiles.clj.